core/src/main/java/org/bitcoinj/core/FullPrunedBlockChain.java

```diff
@@ -164,18 +176,20 @@ public VerificationException call() throws Exception {
         }
     }
 
-    /** Get the {@link Script} from the script bytes or null if it doesn't parse. */
-    @Nullable
+    /**
+     * Get the {@link Script} from the script bytes or null if it doesn't parse.
```

Comment:
This JavaDoc is now out of date. Any reason you changed it? An empty Script object is rather unusual for bitcoinj and I don't think we produce them anywhere else.